torch/nn/quantized/functional.py

```diff
@@ -0,0 +1,5 @@
+from torch._ops import ops
+from torch._C import _add_docstr
+
+relu = ops.quantized.relu
```

Comment:
I think we should import functional when we import `torch.nn.quantized`.

E.g.:
```
In [1]: import torch.nn as nn

In [2]: nn.functional
Out[2]: <module 'torch.nn.functional' from '/data/users/gchanan/_pytorch4a/torch/nn/functional.py'>

In [3]: import torch.nn.quantized as nnq

In [4]: nnq.functional
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-4-fe7fdaf22015> in <module>()
----> 1 nnq.functional

AttributeError: module 'torch.nn.quantized' has no attribute 'functional'
```